README.md

```diff
@@ -29,7 +29,7 @@ The following example shows a typical topology for a KMS instance:
 
 ## Using HPCS instead of Key Protect
 
-This module supports creating key rings and keys for Key Protect or Hyper Protect Crypto Services (HPCS). By default the module creates a Key Protect instance and creates the key rings and keys in that service instance, but this can be modified to use an existing HPCS instance by providing the GUID of your HPCS instance in the `var.existing_key_protect_instance_guid` input variable, and then setting the `var.create_key_protect_instance` input variable to `false`.
+This module supports creating key rings and keys for Key Protect or Hyper Protect Crypto Services (HPCS). By default the module creates a Key Protect instance and creates the key rings and keys in that service instance, but this can be modified to use an existing HPCS instance by providing the GUID of your HPCS instance in the `var.existing_kms_instance_guid` input variable, and then setting the `var.create_key_protect_instance` input variable to `false`. For more information on provisioning an HPCS instance, please see: <https://github.com/terraform-ibm-modules/terraform-ibm-toolkit-hpcs>
```

Comment:
thats not our module. Our module is https://github.com/terraform-ibm-modules/terraform-ibm-hpcs